extensions/amp-story/1.0/amp-story-reaction-poll-binary.css

```diff
@@ -0,0 +1,99 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+ 
+ @import url('https://fonts.googleapis.com/css?family=Poppins:wght@500;700&display=swap');
+
+.i-amphtml-story-reaction-poll-binary-container {
+    font-family: 'Poppins', sans-serif !important;
+    display: flex !important;
+    align-items: stretch !important;
+    justify-content: center !important;
+    height: 90px !important;
+    max-width: 296px !important;
+    border-radius: 16px !important;
+    box-shadow: 0px 2px 4px #000000aa !important;
+    color: var(--reaction-main-color) !important;
+    background: var(--reaction-theme-background) !important;
+    overflow: hidden !important;
+
+    --post-select-scale: 0.5 !important;
+    --post-select-scale-variable: 1;
+}
+
+.i-amphtml-story-reaction-poll-binary-container:not(.i-amphtml-story-reaction-post-selection) {
+    cursor: pointer !important;
+}
+
+[dir="rtl"] .i-amphtml-story-reaction-poll-binary-container {
+    flex-direction: row-reverse;
+}
+
+.i-amphtml-story-reaction-option {
+    display: flex !important;
+    align-items: center !important;
+    justify-content: center !important;
+    text-align: center !important;
+    flex-direction: column !important;
+    flex-grow: 50;
+    flex: 100;
+    overflow: hidden !important;
+}
+
+.i-amphtml-story-reaction-option:hover .i-amphtml-story-reaction-option-title {
+    transform: translateY(-2px);
+}
+
+.i-amphtml-story-reaction-option:first-child {
+    border-right: 1px solid var(--reaction-main-color-translucent) !important;
+}
+
+.i-amphtml-story-reaction-post-selection .i-amphtml-story-reaction-option.i-amphtml-story-reaction-option-selected {
+    background-color: var(--reaction-main-color-transparent) !important;
+}
+
+.i-amphtml-story-reaction-option-title {
+    font-weight: 800 !important;
+    font-size: 28px !important;
```

Comment:
I think these should use `rem` units to make them scalable, instead of `px`

Reply:
(our root font size is variable depending on the size of `amp-story-page`, in pixels)

Reply:
Absolutely! We should switch units to rems in quizzes and polls in a future PR altogether since the styles are still in progress